labview-interop/src/labview.rs

```diff
@@ -12,10 +12,11 @@ use crate::{
     memory::MagicCookie,
 };
 
-/// Represents as UHandle passed by value. Can't use the generic
+/// Represents as UHandle/UPtr passed by value. Can't use the generic
```

Comment:
This represents a single doc comment attached only to the UHandle const. Can you split this to a comment for each?